src/app/pages/portal/portal.component.ts

```diff
@@ -524,6 +524,10 @@ export class PortalComponent implements OnInit, OnDestroy {
     }
   }
 
+  toolToActivateFromWorkspace(event: unknown): void {
+    // @todo...
```

Comment:
@pelord This method is missing in our code base and it's referenced in the Html template. Can you look and fix it?

Reply:
Looks like dead code..

Reply:
Deleted from component and html template.